src/main/java/seedu/duke/command/NoteCommand.java

```diff
@@ -128,7 +128,7 @@ private void parseUserCommand(String command) throws DukeException {
         ArrayList<String> convertedList = new ArrayList<>();
 
         for (String note : notes) { //after splitting up the lines based on semicolons, add them
```

Comment:
Might want to change this comment split based on `

Reply:
Yes, thank you